pkg/helm/helm_cli.go

```diff
@@ -448,7 +448,7 @@ func (h *HelmCLI) UpgradeChart(chart string, releaseName string, ns string, vers
 		args = append(args, "--force")
 	}
 	if timeout != -1 {
-		if h.Binary == "helm3" {
+		if h.BinVersion == V3 {
```

Comment:
Seems ok. Would it make sense to change the other places where we do the check against the binary name of 'helm3' at the same time?